.buildkite/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "../tsconfig.base.json",
```

Comment:
At the time of running the pipeline.ts (or other dynamic pipelines) to emit the PR pipeline, this won't be set up or available - probably should be reverted to the original